build.gradle.kts

```diff
@@ -177,4 +177,8 @@ gradle.projectsEvaluated {
                 }
             }
     }
+
+    tasks.named("build") {
```

Comment:
@romtsn is there a nicer way of declaring the dependency?
Like `build.dependsOn("aggregateJavadocs")`?

Reply:
@romtsn is the Gradle expert, but I guess `dependsOn` does the job.

So `make all` (called in CI) calls `make dryRelease`, maybe we make the `aggregateJavadocs` as part of the Dry release instead?

Reply:
@marandaneto we'd only have to change the task name like this:

```suggestion
    tasks.named("publishToMavenLocal") {
```

Then it would only run on `make dryRelease` not on `./gradlew build`

Reply:
but why do we need to publish it to mavenLocal and not mavenCentral? Just to make the CI pass?

Reply:
Because the makefile says:

```
# do a dry release (like a local deploy)
dryRelease:
	./gradlew publishToMavenLocal --no-daemon --no-parallel
```

Reply:
@romtsn no, it's a way of testing our release generation pipeline.
`publishToMavenLocal` triggers every task needed in order to generate jars, aars, poms, etc.
Uploading to maven central is done thru `craft`.

Reply:
@adinauer maybe changing `dryRelease` only would do:

`./gradlew aggregateJavadocs publishToMavenLocal --no-daemon --no-parallel`

Reply:
I've updated the PR to only change the `Makefile`

Reply:
@adinauer if it works, LGTM.

Reply:
ah ok so only the dryRelease was the problem then it makes sense

Reply:
@romtsn yes, release failed due to Javadoc generation failing. Now we can run the check on every PR to avoid the late detection.

`make all` ran successfully here https://github.com/getsentry/sentry-java/runs/6963622852?check_suite_focus=true

I'll push a problematic commit to test if it actually fails.

Reply:
Fails this PR on problematic javadoc comment, see here https://github.com/getsentry/sentry-java/runs/6963892775?check_suite_focus=true#step:5:7128